app/actions/action_updatePins.js

```diff
@@ -0,0 +1,20 @@
+import { UPDATE_PINS, userData } from '../constants/constants.js';
+
+function updatePins(allPins) {
+  return {
+    type: UPDATE_PINS,
+    payload: allPins
+  };
+}
+
+export default function() {
+ return (dispatch) => {
+  userData.once("value", function(snap) {
+    var fetchedPins = [];
+    snap.forEach(function(savedPin) {
+      fetchedPins.push(savedPin.val());
+    });
+    dispatch(updatePins(fetchedPins));
+  });
+ };
+}
```

Comment:
@asu91 there's an extra line already in the file. Not sure why you can't see it here. The only one w/out the extra line is in AR, which I just fixed